pkg/operator/cluster/types.go

```diff
@@ -78,6 +79,9 @@ type ClusterSpec struct {
 
 	// MonCount sets the mon size
 	MonCount int `json:"monCount"`
+
+	// Resources set resource requests and limits
+	Resources ResourceSpec `json:"resources,omitempty"`
```

Comment:
what if we name it `ResourceRequirements` and `ResourceRequirementsSpec`? `Resource` seems too generic. This would apply to the cluster, mds, and rgw specs

Reply:
nevermind, the name `resources` in the yaml is nice and simple.